lib/task/tools/addSuperuserTask.class.php

```diff
@@ -46,8 +46,6 @@ public function execute($arguments = [], $options = [])
         }
 
         sfContext::createInstance($this->configuration);
-        $databaseManager = new sfDatabaseManager($this->configuration);
-        $conn = $databaseManager->getDatabase('propel')->getConnection();
```

Comment:
Unsure why this exists since this variable is unused. Please let me know if there's a reason for this, and I'll revert this particular change.